pkg/sink/codec/simple/decoder.go

```diff
@@ -168,7 +168,7 @@ func (d *Decoder) NextRowChangedEvent() (*model.RowChangedEvent, error) {
 	if err != nil {
 		return nil, err
 	}
-
+	log.Debug("row changed event assembled", zap.Any("event", event))
```

Comment:
this log is useless

Reply:
It is helpful in debug. I think it is worth to remain it.